src/features/winterStorageAreaList/winterStorageAreaListPage.module.scss

```diff
@@ -0,0 +1,6 @@
+@import 'spacings';
+
+.winterStorageAreaListPage {
```

Comment:
This style seems to be repeated for every "list page". We could extract this as common style to get rid of some `module.scss` files.

Reply:
Same stylings are also applied in some "view pages", could indeed extract to a common file.

Reply:
The same styles were indeed used in _all_ feature pages. I moved them to a common PageContent component